.github/workflows/build-extension-charts.yml

```diff
@@ -83,13 +83,35 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
-          yarn parse-tag-name ${{ inputs.tagged_release }} ${{ github.run_id }} "charts"
+          #!/usr/bin/env bash
+
+          set -eo pipefail
+
+          SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+          BASE_DIR="$(
+            cd $SCRIPT_DIR && cd ../.. &
+            pwd
+          )"
+
+          git clone -b master https://github.com/rancher/dashboard.git dashboard-repo
```

Comment:
I'd suggest we pull this out to a separate step - both steps rely on the repo being cloned, but its cloned in this one and assumed in the next - I'd suggest adding a step above with a comment. I also think we should do a shallow clone/add the appropriate args so we clone as little as possible.